src/markdown/index.ts

```diff
@@ -207,8 +208,10 @@ export const renderPageContent = (
       .processSync(content)
 
     contentHTML = result.toString()
-  } catch (error) {
-    console.error(error)
+  } catch (e) {
+    const error = e as Error
+    toast.error(error.message)
```

Comment:
Maybe error type is string.
```suggestion
    toast.error(error?.message)
```